src/main/sensors/esc_sensor.c

```diff
@@ -1106,21 +1008,30 @@ static void kontronikSensorProcess(timeUs_t currentTimeUs)
             uint32_t crc = kontronikDecodeCRC(kontronikPacketLength - KON_CRC_LENGTH);
             if (calculateCRC32(buffer, kontronikPacketLength - kontronikCrcExclude - KON_CRC_LENGTH) == crc) {
                 uint32_t rpm = buffer[7] << 24 | buffer[6] << 16 | buffer[5] << 8 | buffer[4];
+                int16_t  throttle = (int8_t)buffer[24];
                 uint16_t pwm = buffer[23] << 8 | buffer[22];
                 uint16_t voltage = buffer[9] << 8 | buffer[8];
                 uint16_t current = buffer[11] << 8 | buffer[10];
                 uint16_t capacity = buffer[17] << 8 | buffer[16];
                 int16_t  tempFET = (int8_t)buffer[26];
                 int16_t  tempBEC = (int8_t)buffer[27];
+                uint16_t currBEC = buffer[19] << 8 | buffer[18];
+                uint16_t voltBEC = buffer[21] << 8 | buffer[20];
+                uint16_t status1 = buffer[29] << 8 | buffer[28];
+                uint16_t status2 = buffer[31] << 8 | buffer[30];
```

Comment:
Why not collect the status directly into a uint32_t?

Reply:
Evolutionary leftovers - collecting as 32 done.